docs/pages/contributing/documentation/how-to-contribute.mdx

```diff
@@ -1,168 +1,367 @@
+   [Installing Node.js](https://nodejs.org/en/download/package-manager) 
+   for instructions to download and install Node.js using a package manager.
 
-Navigate to the root of your local clone of the `gravitational/docs` repository
-and run the following command to populate the `content` directory:
+- You have the `yarn` package manager installed.
+   Run `yarn --version` to check whether `yarn` is installed. 
+   If you need to install `yarn`, run the command `brew install yarn` or 
+   `npm install --global yarn` to add it to your local system.
 
-```code
-$ yarn git-update
-```
+## Step 1/5. Set up your local environment
 
-Next, navigate to the directory under `content` that corresponds to the latest version of Teleport.
+To set up a local environment for contributing to Teleport documentation:
 
-```code
-$ cd content/(=teleport.major_version=).x
-```
+1. Open a terminal shell on your computer.
 
-Switch to the master branch (or versioned branch for updates specific to previous versions):
+1. Clone the `gravitational/docs` repository by running the following command:
 
-```code
-git switch master
-```
+    ```code
+    $ git clone https://github.com/gravitational/docs --recurse-submodules
+    ```
 
-Check out a new branch of `gravitational/teleport`.
+    This command clones the main branch for different versions of the documentation.
 
-```code
-$ git switch -c my-branch
-```
+1. Change to the root of the `docs` directory:
+    
+    ```code
+    $ cd docs
+    ```
 
-## Step 2/4 Test your changes
+1. Update the documentation to the latest version from the master branch:
+    
+    ```code
+    $ yarn git-update
+    ```
 
-Run the following command to start the development server:
+1. Install dependencies:
+    
+    ```code
+    $ yarn
+    ```
 
-```code
-$ yarn install
-$ yarn dev
-```
+## Step 2/5. Navigate to a specific version
 
-You can visit the docs site at `localhost:3000/docs`. The development server
-will detect changes to the documentation automatically as you make them and
-refresh the page within the browser.
+Different versions of the documentation are organized into the `content` directory of the 
+`gravitational/docs` repository. Each subdirectory of the `content` directory includes 
+a `git` submodule for a different branch of `gravitational/teleport`.
 
-Validate your changes locally using the provided linters:
+To navigate to a specific version of the documentation:
 
-```code
-# Check your Markdown syntax
-$ yarn markdown-lint
-# markdown-lint, but also checks that external links work
-$ yarn markdown-lint-external-links
-```
+1. View the versions of the documentation listed in the `content` directory by running the following command:
+    
+    ```code
+    $ ls -l content
+    ```
+    
+1. Change to the directory under `content` that contains the version of documentation you want to update.
+    For example, navigate to the directory that corresponds to latest version of Teleport.
```

Comment:
Might be worth noting that you can navigate to the directory corresponding to the current version of Teleport for convenience, regardless of the actual version you're editing, because this is the version that the docs site displays by default.

Not sure if this is too confusing for new contributors, though.